docs/capabilities/portfolio-objectives.md

```diff
@@ -0,0 +1,75 @@
+# Portfolio Objectives 
+
+Being able to set system-specific objectives can help you evaluate your system realistically based on your business context, as well as its current lifecycle phase. However, when you have many systems, setting objectives individually for each of them becomes impractical. It's important to prioritize and set stronger or weaker targets for certain systems, according to their importance for your business. Therefore, it is critical to be able to set objectives for a group of systems. This can be done via portfolio objectives. 
+
+**Portfolio Objective**:  A type of objective that targets a group of systems based on similar metadata, aiming to provide an efficient and user-friendly experience that removes the need to set objectives individually for each system and better prioritization. (General rule) 
+
+
+<img src="../images/portfolio-objectives-main-table.png" width="700" />
+
+
+**System Objective**: A type of objective specific to an individual system within a portfolio that does not follow the general rule. When an objective is set at the system level, the system no longer follows the objective defined at the portfolio level.
+
+
+<img src="../images/portfolio-objectives-system-detail.png" width="700" />
+
+
+## How to define portfolio objectives? 
+
+To define a portfolio objective, the user can click on the "Add Portfolio Objective" button on the top right of the screen:
+
+<img src="../images/portfolio-objectives-add-arrow.png" width="700" />
+
+Upon clicking the button, a pop-up will appear to guide you during the creation of a portfolio objective.
+
+The pop-up will guide you toward configuring the portfolio objective:
+
+<img src="../images/portfolio-objectives-add-popup.png" width="700" />
+
+There will be several dropdown menus available in the pop-up.
+
+
+1. There will be a Capability dropdown, that allows the user to select the specific capability for which the portfolio objective will be defined.
+
+2. Then, the Type dropdown, allows the user to select a given objective type that is part of the capability selected in the previous step. In the example, we see that the Test Code Ratio is set as a type. This is due to the fact that the "Test Code Ratio" objective is a type of objective that belongs to the Maintainability capability.
+
+3. Finally, the last step allows for defining the actual value that we want our portfolio objective to be. 
+This final field can be of two distinct types: either a free-text field, in the case of numeric objectives, or a dropdown containing a series of pre-defined values, that will be the allowed values for the given objective type.
+
+Note also the indicator that the objective will apply to all systems in a given portfolio.
```

Comment:
"in a given portfolio" -> "your portfolio".

Reply:
Thanks, corrected!